tests/unit/components/heatmap/HeatMap.spec.js

```diff
@@ -81,18 +81,18 @@ describe("HeatMap.vue", () => {
     const init_heatmap_values = {
       "Twitch Frequency": {
         data: [
-          new Array(MIN_NUM_DATAPOINTS_FOR_MEAN).fill(1),
-          new Array(MIN_NUM_DATAPOINTS_FOR_MEAN).fill(2),
-          new Array(MIN_NUM_DATAPOINTS_FOR_MEAN).fill(4),
-          new Array(MIN_NUM_DATAPOINTS_FOR_MEAN).fill(5),
+          new Array(MAX_NUM_DATAPOINTS_FOR_MEAN).fill(1),
+          new Array(MAX_NUM_DATAPOINTS_FOR_MEAN).fill(2),
+          new Array(MAX_NUM_DATAPOINTS_FOR_MEAN).fill(4),
+          new Array(MAX_NUM_DATAPOINTS_FOR_MEAN).fill(5),
         ],
       },
       "Twitch Force": {
         data: [
-          new Array(MIN_NUM_DATAPOINTS_FOR_MEAN).fill(1),
-          new Array(MIN_NUM_DATAPOINTS_FOR_MEAN).fill(2),
-          new Array(MIN_NUM_DATAPOINTS_FOR_MEAN).fill(4),
-          new Array(MIN_NUM_DATAPOINTS_FOR_MEAN).fill(5),
+          new Array(MAX_NUM_DATAPOINTS_FOR_MEAN).fill(1),
+          new Array(MAX_NUM_DATAPOINTS_FOR_MEAN).fill(2),
+          new Array(MAX_NUM_DATAPOINTS_FOR_MEAN).fill(4),
+          new Array(MAX_NUM_DATAPOINTS_FOR_MEAN).fill(5),
```

Comment:
We should add one more test like this one where less than 5 data points are present